src/media/js/forms_transonic.js

```diff
@@ -143,7 +143,7 @@ define('forms_transonic',
             $region_feed.find('.feed-element').each(function(i, feed_element) {
                 data[region].push([feed_element.getAttribute('data-type'),
                                    feed_element.getAttribute('data-id')]);
-            });
+            }); // I think this has a bug in it because of regions...
```

Comment:
I switched the for loop to a `forEach`. Hopefully that doesn't cause any problems.